livekit-agents/livekit/agents/llm/chat_context.py

```diff
@@ -26,11 +26,40 @@
 
 @dataclass
 class ChatImage:
+    """
+    ChatImage is used to input images into the ChatContext on supported LLM providers / plugins.
+
+    You may need to consult your LLM provider's documentation on supported URL types.
+
+    ```python
+    # With a VideoFrame, which will be automatically converted a data URL internally
```

Comment:
Is this missing a word?

```suggestion
    # With a VideoFrame, which will be automatically converted to a data URL internally
```